docs/website/docs/general-usage/dashboard.md

```diff
@@ -73,43 +73,83 @@ The following sections are available:
+The overview section gives you a general sense of the state of your pipeline and will also display exception information if the last run of your pipeline failed. Review last executed to ensure the pipeline is running on the expected schedule or has recently completed a load.
 
 ![Pipeline overview](https://storage.googleapis.com/dlt-blog-images/dashboard-pipeline-overview.png)
 
+The field **last executed** tells you the timestamp of the last successful pipeline run which is beneficial for debugging, **credentials** contains the full path or connection string used to connect to the destination and **schemas** lists the schemas associated with the pipeline.
+
+#### Remote state
+
+This section queries and displays the state information that has been successfully committed to the **destination**, confirming what is stored outside the local working directory.
+
+* The `state_version` shown here should generally match the `state_version` in the Pipeline Overview if the last run was successful.
+* Click `Load and show remote state` to query the destination and retrieve this information.
+
 ### Schema explorer
 
-The schema explorer allows you to dive deep into your pipeline's schemas and see all tables, columns, and hints.
+The schema browser allows you to dive deep into your pipeline's schemas and see all tables, columns, and type hints. This view is based on the dlt schema, the internal blueprint used to create and update tables in your destination.
+
+When you're loading your data for the first time, we suggest checking the following:
+
+1. Click `Show child tables` to ensure that nested data is unnested successfully and at the required level.
+2. Inspect the **type hints** specifically, making sure that timestamps and large numbers are captured in the right way.
+3. Raw Schema as YAML can be exported and stored in your git repository as a reference for the dataset schema.
 
 ![Schema explorer](https://storage.googleapis.com/dlt-blog-images/dashboard-schema-explorer.png)
 
 ### Dataset Browser
 
-The dataset browser allows you to query data from your destination and inspect the state of each of your resources, which contain information about their incremental status and other custom state values.
+The dataset browser allows you to query data from your destination and inspect the state of each of your resources, which contain information about their incremental status and other custom state values. Click the `Load row counts` button to refresh the record counts for all tables based on the current destination data.
 
 ![Dataset Browser](https://storage.googleapis.com/dlt-blog-images/dashboard-dataset-browser.png)
 
 ### Querying your Dataset
 
-This is what a query on the fruitshop dataset looks like:
+Here you can query your data to analyze it. This is what a query on the fruitshop dataset looks like:
```

Comment:
Please, add note that it's working with any destinations even with the files, Iceberg tables, Delta tables, with remote and local destinations